src/autocomplete/use-autocomplete.ts

```diff
@@ -131,15 +117,25 @@ export const useAutocomplete = <I>({
 		query,
 		textual,
 		value,
-		values$,
-		items$: useMemo(() => {
-			if (!active || (hideEmpty && empty)) {
-				return EMPTY$;
-			}
-			return query && !external
```

Comment:
`external` was supposed to allow us to skip local filtering when BE does it.

Reply:
Well... we don't use it at the moment. We can add it back later when the need arises.

Reply:
Don't we use this in omnitable ?